package-lock.json

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
You have both a yarn.lock and a pack-lock file. That means you used both npm and yarn in the project. In the future try to stick with one and delete the other.